### ISSUE TITLE
EntityRegainHealthEvent error with Spartan

### ISSUE THREAD

**Issue by CryLegende** [kernitus/BukkitOldCombatMechanics#609]
Hello, since the latest builds, I have an error from OCM in my console, related to Spartan

PaperSpigot 1.12.2
OCM build 102 (latest build)
Config: https://pastebin.com/raw/iP7FXh21
Error: https://pastebin.com/raw/Fyu7cFxs

Thank you for your help

**Comment by kernitus** [comment]
I don't think we changed anything relevant to Spartan anticheat in the past few commits. However I have updated the SpartanAPI dependency, in case Spartan itself has changed something, if you could please test the [latest build](https://github.com/kernitus/BukkitOldCombatMechanics/suites/6334218102/artifacts/227989478).

**Comment by kernitus** [comment]
@CryLegende got a [new build](https://ci.rayzr.dev/job/OldCombatMechanics/105/) with a different version if you can please try it
